### ISSUE TITLE
Error during upgrade

### ISSUE THREAD

**Issue by mimmus** [coreroller/coreroller#38]
I'm trying to upgrade a server from 1185.5.0 to 1235.12.0 but I'm getting always an error, this is the log:

http://pastebin.com/eexsGRXU

1235.12.0 was packaged by me, because I want to halt my servers to this version. Peraphs a mistake during manual package creation?

Any help?

**Comment by tegioz** [comment]
I've taken a quick look at the logs and I've seen a timeout error getting the download package (from https://update.release.core-os.net/amd64-usr/1235.12.0/update.gz). Have your instances access to the Internet? While it's possible to host the packages using CoreRoller, by default the official packages urls are used, which means your instances need access to the Internet to fetch them.

What do you mean by "was packaged by me"? Did you create the package entry using the CoreRoller UI or did you build your own modified version of CoreOS (generating your own update.gz file)? Adding packages manually to CoreRoller is ok, you just have to make sure the hashes are computed correctly but other than that it should work fine.

**Comment by mimmus** [comment]
First of all, thank you for your support (I'm asking many questions in these days).

Yes, it was a problem accessing the Internet.
I removed the proxy because it was cause of issue accessing the internal Coreroller server and I thought that packages were hosted by the Coreroller itself. I solved setting proxy again and using NO_PROXY for internal server.

Than I had other issues on the CoreOS servers because they have a previous update pending (with reboot needed), I had to reset its update status using:
 update_engine_client -reset_status

I packaged an older CoreOS version using CoreRoller UI, that was OK.

Now I have a fully working CoreRoller setup!

Thanks again
